20070830, Stuart Hughes <stuarth at freescale dot com>

Fake vt102 as vt100, they're almost the same and I don't expect
anyone has a real vt102.  This was needed as telnet presents
as a vt102 by default and this was preventing minicom from starting

diff --exclude CVS -uNr termcap-mini-1.2/termcap.mini termcap-mini-1.2.modified/termcap.mini
--- termcap-mini-1.2/termcap.mini	2007-08-03 12:08:25.000000000 +0100
+++ termcap-mini-1.2.modified/termcap.mini	2007-08-30 11:59:42.000000000 +0100
@@ -43,7 +43,7 @@
 	:sf=\E[S:so=\E[7m:sr=\E[T:ta=^I:te=\E[m:ti=\E[m:up=\E[A:km:pt:\
 	:AL=\E[%dL:DL=\E[%dM:SF=\E[%dS:SR=\E[%dT:al=\E[L:dl=\E[M:
 
-vt100|vt100-am|dec vt100 (w/advanced video):\
+vt102|vt100|vt100-am|dec vt100 (w/advanced video):\
 	:am:bs:ms:xn:xo:\
 	:co#80:it#8:li#24:vt#3:\
 	:@8=\EOM:DO=\E[%dB:K1=\EOq:K2=\EOr:K3=\EOs:K4=\EOp:K5=\EOn:\
